### PR TITLE
Change scalability test configs

### PR DESCRIPTION
**Pull request by gmarek**
cc @davidopp @spxtr

### CODE REVIEW

**Comment by spxtr** [comment]
Once this goes in, ping me and I can delete the old jobs. We don't yet do that automatically.

**Comment by gmarek** [comment]
@k8s-bot unit test this issue: #IGNORE
`curl: (18) transfer closed with 6596924 bytes remaining to read`

**Comment by gmarek** [comment]
Munger changes are already in. @wojtek-t PTAL

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml [hunk 1]
```diff
@@ -518,7 +524,8 @@
     name: kubernetes-e2e-gce-enormous-cluster
     test-owner: 'gmarek'
     emails: 'gmarek@google.com'
-    cron-string: 'H 8 * * *'
+    # Run only on Sunday at 8 am MTV time.
+    cron-string: 'H 8 * * 0'
```
Comment (wojtek-t) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml]: Let's run it twice a week for some more time

Comment (wojtek-t) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml]: Like 2 more weeks - we have just stabilized it and I would like to confirm that it works.

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml [hunk 2]
```diff
@@ -110,7 +110,7 @@
             description: 'Run Density test on Kubemark in very large cluster. Currently only scheduled to run every 6 hours so as not to waste too many resources.'
             # 12h - load tests take really, really, really long time.
             timeout: 720
-            cron-string: 'H H/8 * * *'
+            cron-string: 'H H/12 * * *'
```
Comment (wojtek-t) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-kubemark.yaml]: Please update description above.

**Comment by gmarek** [comment]
@wojtek-t PTAL

cc @spxtr - munger is reconfigured, so we're going to merge this PR today.

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**